llama_index/query_engine/multi_modal.py

```diff
@@ -108,6 +113,27 @@ def synthesize(
             metadata={"text_nodes": text_nodes, "image_nodes": image_nodes},
         )
 
+    def image_synthesize(
```

Comment:
I think image_synthesize is confusing me with normal synthesize -- maybe we can rename to def _get_ response_with_images() and change the typing to be List[ImageNode] ?